tests/issues/GH4796.test.ts

```diff
@@ -41,6 +41,9 @@ test('4796', async () => {
   const u1 = orm.em.create(User, { options: ['\\'] });
   await orm.em.flush();
 
+  const ud = await orm.em.findOne(User, { id: u1.id });
```

Comment:
this will help, its the same as clearing the context, as forks are always clean

```suggestion
  const ud = await orm.em.fork().findOne(User, { id: u1.id });
```